fixtures/nodejs-hybrid-app/package.json

```diff
@@ -9,7 +9,7 @@
 	},
 	"devDependencies": {
 		"@cloudflare/workers-tsconfig": "workspace:*",
-		"@cloudflare/workers-types": "^4.20240222.0",
+		"@cloudflare/workers-types": "^4.20240605.0",
```

Comment:
There was no actual release of workers-types this week.
So generally there are no updates to make.
This particular one got missed last week.